src/Dumplie/Application/Command/Customer/AddToCart.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare (strict_types = 1);
+
+namespace Dumplie\Application\Command\Customer;
+
+final class AddToCart
+{
+    /**
+     * @var string
+     */
+    private $sku;
+
+    /**
+     * @var int
+     */
+    private $quantity;
+
+    /**
+     * @var string
+     */
+    private $cartId;
+
+    /**
+     * AddToCart constructor.
+     * @param string $sku
+     * @param int $quantity
+     * @param string $cartId
+     */
+    public function __construct(string $sku, int $quantity, string $cartId)
+    {
+        $this->sku = $sku;
+        $this->quantity = $quantity;
+        $this->cartId = $cartId;
+    }
+
+    /**
+     * @return string
+     */
+    public function sku() : string
+    {
+        return $this->sku;
+    }
+
+    /**
+     * @return int
+     */
+    public function quantity() : int
+    {
+        return $this->quantity;
+    }
+
+    /**
+     * @return string
+     */
+    public function cartId() : string
```

Comment:
**get**CardId() convention?

1. fields = stuff, methods=messages => messages should have verbs in them to sound like u "ask an object for something"
2. It is commonly used convention, understood by everybody
3. There are tools that require such convention, [like PropertyAccess Component](https://symfony.com/doc/current/components/property_access/introduction.html#using-getters)